sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderClientTest.java

```diff
@@ -301,6 +303,89 @@ void scheduleMessageWithTransaction() {
         verify(asyncSender).scheduleMessage(testData, scheduledEnqueueTime, transactionContext);
     }
 
+    /**
```

Comment:
Can you run all the live tests? I'm afraid we'll run into dirty tests and conflicts like what Yijun experienced